### PR TITLE
Add proc.username field documentation for Topbeat

### PR DESCRIPTION
**Pull request by andrewkroh**

### CODE REVIEW

File: topbeat/docs/fields.asciidoc [hunk 1]
```diff
@@ -328,6 +328,27 @@ type: float
 The percentage of used swap memory.
 
 
+==== swap.actual_used
```
Comment (andrewkroh) [inline topbeat/docs/fields.asciidoc]: These showed up when I regenerated the docs.

Comment (monicasarbu) [inline topbeat/docs/fields.asciidoc]: Good catch. It looks like somehow they are not deleted from etc/fields.yml. I checked gosigar code and the Swap structure contains only the information: Total, Used, Free. Could you please delete the rest of fields in this PR? Thanks

**Comment by monicasarbu** [comment]
LGTM

File: topbeat/docs/fields.asciidoc (hunk 1 above)
Comment (andrewkroh) [inline topbeat/docs/fields.asciidoc]: Yes, I will delete them today.

Comment (andrewkroh) [inline topbeat/docs/fields.asciidoc]: I removed the fields and regenerated the asciidoc.

**Comment by owulff** [comment]
Guys, you dropped this feature again. We're looking for this for months now. Initally dropped from beta4, then GA 1.0 and now again from 1.1 :-(

**Comment by tsg** [comment]
Apologies, the feature is finally implemented now but was merged too late for 1.1. You can use the nightlies if you want, and we'll make sure it will be in 1.2, which is scheduled 6 weeks from now.

If 1.2 is too far away and nightlies are too risky, we'll also have a 2.0.0-beta1 release in a couple of weeks from now, which should provide some middle ground between the two options.

**Pull request merged**